src/main/java/com/inqoo/trainingservice/app/subcategory/SubCategoryConverter.java

```diff
@@ -0,0 +1,22 @@
+package com.inqoo.trainingservice.app.subcategory;
+
+import org.springframework.stereotype.Component;
+
+@Component
```

Comment:
Usualy util classes like converters doesn't have to be Spring beans. Please consider construct it by yourself or change to static calls

Reply:
done